src/client/components/CampaignBadge/CampaignBadge.tsx

```diff
@@ -7,15 +7,16 @@ import { useAppliedCampaignNameQuery } from 'data/graphql'
 import { useTextKeys } from 'utils/textKeys'
 import { useCurrentLocale } from 'l10n/useCurrentLocale'
 
-const CampaignBadgeStyled = styled.div`
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-start;
+/* https://coryrylan.com/blog/css-gap-space-with-flexbox */
+const InlineFlexGap = styled.div`
+  --gap: 0.5rem;
+  display: inline-flex;
   flex-wrap: wrap;
-  align-items: flex-start;
+  margin: calc(-1 * var(--gap)) 0 0 calc(-1 * var(--gap));
```

Comment:
nit: Can be a bit hard to read long shorthands IMO and no need to set bottom and right margins :)
```suggestion
  margin-top: calc(-1 * var(--gap));
  margin-left: calc(-1 * var(--gap));
```